src/http-api/error-handler.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+const Hoek = require('hoek')
+
+module.exports = (server) => {
```

Comment:
@dignifiedquire - thanks for adding/overhauling this...I'll add myself a chore to start dropping swapping this in throughout.